src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -266,7 +263,7 @@ abstract class FixupSession(
 
     for (op in workspaceEditParams.operations) {
 
-      op.uri?.let { createAndSwitchFileIfNeeded(it) }
+      op.uri?.let { updateEditorIfNeeded(it) }
```

Comment:
so this is no longer a fixup session's responsibility to create the files... the other endpoint handles that now

Reply:
Yep :) Now it is done how I should do it in the first place if I knew better :)